docs/installation-and-operations/installation/packaged/README.md

```diff
@@ -521,6 +527,22 @@ OpenProject heavily relies on caching, which is why the wizard suggests you to i
 
 ![06-cache](06-cache.png)
 
+## Step 7: Host name and Protocol (only when Apache installation was skipped)
```

Comment:
Thank you for adding documentation for this. It got me confused when I was looking up the code to find where the SSL setting is actually set (because it's not here but in the pkgr apache addon).